modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -51,25 +63,26 @@ def validate_form_526_submission_values(target_veteran)
       def validate_form_526_change_of_address
         return if form_attributes['changeOfAddress'].blank?
 
-        validate_form_526_change_of_address_required_fields
-        validate_form_526_change_of_address_beginning_date
-        validate_form_526_change_of_address_ending_date
+        change_of_address = form_attributes['changeOfAddress']
+
+        validate_form_526_address_type(change_of_address, '/changeOfAddress/')
+        validate_form_526_change_of_address_required_fields(change_of_address)
+        validate_form_526_change_of_address_beginning_date(change_of_address)
+        validate_form_526_change_of_address_ending_date(change_of_address)
```

Comment:
These last 3 lines were just a refactor since I had to add the variable above on line 66 for the type chek I just sent it in for each of those instead of leaving the assignment of the same thing in each of those